routes/store.mjs

```diff
@@ -33,6 +33,7 @@ router.post(
       return;
     }
 
+    inviteIfNecessary(username);
```

Comment:
This should be `await`ed right?

Reply:
Hmmm... Yes. I don't know if it's strictly required since we're not using the return value, but better safe than sorry, probably.